ext/date/php_date.c

```diff
@@ -60,6 +60,25 @@ static inline long long php_date_llabs( long long i ) { return i >= 0 ? i : -i;
 #endif
 #endif
 
+time_t php_time(time_t *loc) {
```

Comment:
IMHO we should drop the `loc` argument here:

> The  tloc  argument  is  obsolescent  and should always be NULL in new code.  When tloc is NULL, the call cannot fail.

Reply:
done